tests/config/driverconfig/powerflex/v2.6.0/controller.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: <DriverDefaultReleaseName>-controller
-  namespace: <DriverDefaultReleaseNamespace>
+  namespace: <DriverDefaultReleaseName>
```

Comment:
Should this not be DriverDefaultReleaseNamespace?

Reply:
Same comment as above